aws/resource_aws_gamelift_fleet_test.go

```diff
@@ -0,0 +1,625 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/gamelift"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+const testAccGameliftFleetPrefix = "tf_acc_fleet_"
+
+func init() {
+	resource.AddTestSweepers("aws_gamelift_fleet", &resource.Sweeper{
+		Name: "aws_gamelift_fleet",
+		Dependencies: []string{
+			"aws_gamelift_build",
+		},
+		F: testSweepGameliftFleets,
+	})
+}
+
+func testSweepGameliftFleets(region string) error {
+	client, err := sharedClientForRegion(region)
+	if err != nil {
+		return fmt.Errorf("error getting client: %s", err)
+	}
+	conn := client.(*AWSClient).gameliftconn
+
+	resp, err := conn.ListFleets(&gamelift.ListFleetsInput{})
+	if err != nil {
+		return fmt.Errorf("Error listing Gamelift Fleets: %s", err)
+	}
+
+	if len(resp.FleetIds) == 0 {
+		log.Print("[DEBUG] No Gamelift Fleets to sweep")
+		return nil
+	}
+
+	out, err := conn.DescribeFleetAttributes(&gamelift.DescribeFleetAttributesInput{
```

Comment:
We should probably check `NextToken` here for good measure like `_getGameliftFleetFailures`

Reply:
So the assumption I'm making here is that since we're performing ID-based lookup we're always going to get `1` or `0` results, so pagination should never be necessary?

Reply:
Sorry I meant to put this on `conn.ListFleets` above 😞  (just so this goes and sweeps everything there are additional results). Really just a minor nitpick though.